scripts/linters/js_ts_linter.py

```diff
@@ -46,6 +47,12 @@
 # pylint: enable=wrong-import-order
 # pylint: enable=wrong-import-position
 
+EXCLUDE_ANY_TYPE_FILES_PATH = os.path.join(
```

Comment:
I think that `FILES_EXCLUDED_FROM_ANY_TYPE_CHECK_PATH` would be better.

Reply:
Done